data/rulesets.js

```diff
@@ -714,9 +714,9 @@ let BattleFormats = {
 			return -typeMod;
 		},
 	},
-	natdex: {
+	natdexrule: {
 		effectType: 'Rule',
-		name: 'NatDex',
+		name: 'NatDex Rule',
```

Comment:
Did this really need to be renamed?

Reply:
I made the decision to rename this instead of remove the alias since I figured "natdex" would be more useful as a format alias than as a ruleset name.

Reply:
We've never called anything "rule", though. Why not NatDex Clause or NatDex Mod?

Reply:
No specific reason other than I thought that would be fine, I don't mind it being changed.